tests/phpunit/CRM/Utils/AddressTest.php

```diff
@@ -12,10 +12,10 @@ public function testAddressFormat(): void {
       'last_name' => 'mouse',
       'contact_type' => 'Individual',
     ]);
-    $address = $this->callAPISuccess('address', 'create', [
+    $address = $this->callAPISuccess('Address', 'create', [
       'street_address' => '1 Happy Place',
       'city' => 'Miami',
-      'state_province' => 'Flordia',
+      'state_province' => 'Florida',
```

Comment:
I suppose it could be interesting ("What happens when the `state_province` is wonky?")... but it's clearly not the focus of the test... 

👍

Reply:
lol yes